packages/element-plus/package.json

```diff
@@ -56,6 +56,7 @@
     "normalize-wheel-es": "^1.1.1"
   },
   "devDependencies": {
+    "@ctrl/tinycolor": "^3.4.0",
```

Comment:
We should move it to `dependencies` @YunYouJun

Reply:
Ok, i create https://github.com/element-plus/element-plus/pull/4842 for it.